archive/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db              import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.utils           import timezone
+
+from user.models    import User
+from content.models import Content
+
+
+class Rating(models.Model):
+     user        = models.ForeignKey(User, on_delete = models.CASCADE, related_name='rated_movie')
+     content     = models.ForeignKey(Content, on_delete = models.CASCADE, related_name='rating_user')
+     rating      = models.FloatField(validators=[MinValueValidator(0.5), MaxValueValidator(5.0)])
+     watch_date  = models.DateField(null=True)
+     create_time = models.DateTimeField(default = timezone.now)
+     class Meta:
+         db_table = 'ratings'
+
+
+class ArchiveType(models.Model):
+    name = models.CharField(max_length=10)
+    class Meta:
+        db_table = 'archivetypes'
```

Comment:
```suggestion
        db_table = 'archive_types'
```
테이블 이름에서 간격이 필요한 경우에는 언더바 (`_`)를 사용해주시면 됩니다 :)

Reply:
내용 수정 완료했습니다!